LICENSE

```diff
@@ -175,3 +175,28 @@
       of your accepting any such warranty or additional liability.
 
    END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```

Comment:
Added PR to change the template:
https://github.com/intelsdi-x/snap-pluginsync/pull/20